README.md

```diff
@@ -95,6 +95,12 @@ Related reports and publications can be found at the
 
 [TLA+]: http://lamport.azurewebsites.net/tla/tla.html
 [Microsoft Z3]: https://github.com/Z3Prover/z3
+---
+
+Apalache is developed at [Informal Systems](https://informal.systems/).
+
+With additional funding from [![Vienna Business Agency Logo](TODO)](https://viennabusinessagency.at/).
```

Comment:
@konnov do you know what logo we're meant to use here?

Reply:
The only logo I have found is this one: https://viennabusinessagency.at/fileadmin/user_upload/Ad_Logo/Logo_Dt__Wirtschaftsagentur_StadtWien.zip

Is it too large?

Reply:
That'll work! Thanks :)